docs/sphinx/source/modelchain.rst

```diff
@@ -390,6 +411,72 @@ shown below.
 
     mc.infer_ac_model??
 
+
+ModelChain for a PVSystem with multiple Arrays
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The PVSystem can represent a PV system with a single array of modules, or
+with [multiple arrays](#PVSystem and Arrays). The same models are applied to
```

Comment:
Anyone know how to link to a heading in a different file (pvsystem.rst, in this case)?